src/elements/tiles/Tiles.scss

```diff
@@ -16,6 +16,10 @@ novo-tiles {
         cursor: pointer;
         align-items: center;
 
+        &.disabled {
+            opacity: .5;
```

Comment:
ah, i see what you are saying. sounds like we should rename the variable instead then to make it more clear. ill think of something :thinking: